docs/welcome.mdx

```diff
@@ -48,8 +48,7 @@ low-latency, resilient, and secure service that's easy to use and set up.
 
 In short: Ory Cloud is the most convenient and straightforward way to run Ory.
 Sign up at [console.ory.sh](https://console.ory.sh/registration) for an account
-and create a free developer project. Read more about the
-[Developer Plan here](./cloud/developer).
+and create a free developer project.
```

Comment:
did we remove developer (startup) plan description document?

Reply:
yes I also removed it because it was quite outdated (list of features). 
I suggest to create a page on the website to explain pricing and things like that in more detail.

Reply:
ok, thx for the info :)